.npmignore

```diff
@@ -2,9 +2,9 @@
 # npm-debug.log and node_modules/ are ignored by default.
 # See https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package
 
-# Ignore all dot files except for .lounge_home
+# Ignore all dot files except for .thelounge_home
 .*
-!.lounge_home
+!.thelounge_home
```

Comment:
Content of this file is still `~/.lounge`. I couldn't find a way to change this to `~/.thelounge` without introducing any breaking changes.
Unless some has a better idea, I guess the switch will be made in actual v3 release.